src/template-tags/tickets.php

```diff
@@ -733,7 +733,7 @@ function tribe_tickets_get_capacity( $post ) {
 
 	// When we have a legacy ticket we migrate it
 	if ( ! in_array( $post->post_type, $event_types ) && tribe( 'tickets.version' )->is_legacy( $post->ID ) ) {
-		return tribe( 'tickets.handler' )->filter_capacity_support( null, $post->ID, $key );
+		return (int) tribe( 'tickets.handler' )->filter_capacity_support( null, $post->ID, $key );
```

Comment:
I generally don't like complicated one line returns, such as a function that calls a method that gets cast. Maybe consider setting a var one line above and then returning that var casted? Not a big deal, just improves readability.

Reply:
True that. Will change.